build/multilocale.js

```diff
@@ -67,7 +67,7 @@ function L10nManager(gaiaDir,
                                  self.official === '1' ?
                                    'official' : 'unofficial',
                                  utils.basename(resURL));
-        isOfficialBranding = true;
+        isOfficialBranding = self.official === '1';
```

Comment:
Nit: you can probably move this above `realURL` line and use the new value of `isOfficalBranding` in the ternary if.